fastlane/metadata/ar-SA/name.txt

```diff
@@ -1 +1 @@
-WordPress
+ووردبريس – مُنشئ مواقع الويب
```

Comment:
I "fixed" this at the GlotPress source. I'm not actually sure the way I wrote it is correct, but GlotPress marks my changed as waiting, so someone will look at it soon, hopefully.

Before:

<img width="668" alt="image" src="https://user-images.githubusercontent.com/1218433/158316237-75874872-34b4-4546-86c1-8fb50d4647b2.png">

After:

<img width="960" alt="image" src="https://user-images.githubusercontent.com/1218433/158316494-b2459f5c-f9bc-49ef-8d77-d0368569ac1b.png">